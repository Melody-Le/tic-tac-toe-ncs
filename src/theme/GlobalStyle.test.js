import "jest-styled-components";
import GlobalStyle from "./GlobalStyle";
import renderer from "react-test-renderer";

test("should have global style", () => {
  renderer.create(<GlobalStyle />);
  expect(document.head).toMatchSnapshot();
});
