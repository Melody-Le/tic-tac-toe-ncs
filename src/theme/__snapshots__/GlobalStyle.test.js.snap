// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have global style 1`] = `
<head>
  <style
    data-styled="active"
    data-styled-version="6.1.1"
  />
</head>
`;
